colors/primary.vim

```diff
@@ -25,8 +25,9 @@ if exists('syntax_on')
 endif
 let g:colors_name = 'primary'
 let s:disable_italic = get(g:,'colorscheme_primary_disable_italic', 0)
+let s:enable_bg_transparency = get(g:,'colorscheme_primary_bg_transparency', 0)
```

Comment:
For the variable name here, I'm wondering, have you seen any other colorschemes do similar, and do they have any existing variable name we could adopt as a convention? If not, this is fine as-is.

Reply:
Hi,

There seems to be no convention about this option, for example:

gruvbox: 
g:gruvbox_transparent_bg
https://github.com/morhetz/gruvbox/wiki/Configuration#ggruvbox_transparent_bg

solarized: 
g:solarized_termtrans
https://github.com/altercation/vim-colors-solarized

So, can we use this `g:colorscheme_primary_enable_transparent_bg` ?

Reply:
Yep, this works for me, and the cross-references are really helpful, thanks!